index.bs

```diff
@@ -2132,26 +2132,29 @@ and a [=header list=] |newHeaders| (defaults to an empty [=list=]):
 
 This algorithm constructs a [=request=] and attempts to deliver it to a [=suitable origin=].
 
+To <dfn>remove a report from the cache </dfn> given an [=attribution report=] |report|:
+
+1. If |report| is an:
+    <dl class="switch">
+    <dt>[=event-level report=]</dt>
+    <dd>[=Queue a task=] to [=list/remove=] |report| from the [=event-level report cache=].</dd>
+
+    <dt>[=aggregatable report=]</dt>
+    <dd>[=Queue a task=] to [=list/remove=] |report| from the [=aggregatable report cache=].</dd>
+    </dl>
+
 To <dfn>attempt to deliver a report</dfn> given an [=attribution report=] |report|, run the following steps:
 
 1. Let |url| be the result of executing [=generate an attribution report URL=] on |report|.
 1. Let |data| be the result of executing [=serialize an attribution report=] on |report|.
-1. If |data| is an error, return.
+1. If |data| is an error, run [=remove a report from the cache=] and return.
```

Comment:
nit: should this be "run [=remove a report from the cache=] with |report| and return"?